pulp_2to3_migration/app/plugin/rpm/yum_repo_metadata.py

```diff
@@ -0,0 +1,13 @@
+from mongoengine.queryset.visitor import Q as mongo_Q
```

Comment:
my only suggestion would be to move this to plugins/rpm/utils.py instead

Reply:
@ipanova , did you mean xml_utils.py? or do you suggest to rename this file to utils.py?

Reply:
My suggestions was to rename the file to utils.py